README.md

```diff
@@ -1 +1,172 @@
-# notification-service
\ No newline at end of file
+# Konflux Notification Service
+
+The Notification Service is a controller that sends push pipelineruns results to 
+[AWS SNS service](https://aws.amazon.com/sns/).
+It watches for `push pipelineruns`, extract the results from pipelineruns that ended successfully 
+and sends them to a topic defined in `AWS SNS`.
+
+Secrets and environment variables are needed to configure the `AWS SNS`
+
+## AWS credentials
+
+`AWS Access key id` and `AWS secret access key` are needed to sign requests to AWS.
+The keys can be provided as a secret (prefered option) or as environment variables
+
+### AWS credentials as a secret
+
+The prefered way to supply the credentials is to create a secret containing the credentials in the format:
+
+key name: `credentials`  
+value: 
+```
+[default]
+aws_access_key_id=<AWS_ACCESS_KEY_ID>
+aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
+```
```

Comment:
The format is not clear to me. Is the stuff starting with default the content of `value`?

Reply:
I added a reference to AWS documentation, I hope its clearer

Reply:
I'm not familiar with the format. The lines above `[default]`, are those inside the file as well? If so, then need to move the those quotes higher.

Reply:
I looked at the reference now, the snippet looks good, but what's that part above `[default]`?

Reply:
According to the reference:
`The [default] heading defines credentials for the default profile, which the SDK will use unless you configure it to use another profile.`

Reply:
I updated the description a bit.